### PR TITLE
Adding a Caption and Timestamp to all elements

### PR DESCRIPTION
**Pull request by a1ly404**
EE-689
Added in the Caption and Timestamp field on the editElementScreen and the previewElementScreen. Also added in the redux-debbuger so we can use the Redux devtools.
Needed to add in react-native-screens otherwise building failed.

### CODE REVIEW

File: screens/PreviewElementScreen.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ class PreviewElementScreen extends React.Component {
 
   render() {
     let { imageUri } = this.state;
-    let { caption } = this.state.params.item;
-    let { geo } = this.state.params.item;
+    let { caption, timestamp, geo } = this.state.params.item;
```
Comment (KitArmstrong) [inline screens/PreviewElementScreen.js:65]: Thanks for cleaning up.

File: js/renderFunctions.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+    Text,
+    TouchableHighlight,
+    View,
+  } from 'react-native';
+import { Image } from 'react-native-elements'
+export const renderTouchables = (key, imageSource, p, styles) => {
```
Comment (KitArmstrong) [inline js/renderFunctions.js]: What is "p"? Can we use a more descriptive param?

**Review by KitArmstrong — CHANGES_REQUESTED** [review]

File: js/renderFunctions.js (hunk 2 above)
Comment (a1ly404) [reply js/renderFunctions.js]: changed to item

**Review by marklise — APPROVED** [review]
LGTM

**Review by KitArmstrong — APPROVED** [review]

**Pull request merged**